### PR TITLE
invalidate ownership cleanliness flags when reassigning cases

### PR DESCRIPTION
**Pull request by czue**
http://manage.dimagi.com/default.asp?187798

@snopoke not the cleanest, but it works. any thoughts on improvements?

cc @orangejenny 

definitely wait for tests

### CODE REVIEW

File: corehq/ex-submodules/casexml/apps/phone/tests/test_sync_mode.py [hunk 1]
```diff
@@ -1,3 +1,4 @@
+from unittest import skip
```
Comment (dimagimon) [inline corehq/ex-submodules/casexml/apps/phone/tests/test_sync_mode.py]: F401 'skip' imported but unused

File: corehq/form_processor/backends/sql/processor.py [hunk 2]
```diff
@@ -211,18 +212,23 @@ def get_cases_from_forms(case_db, xforms):
             rebuild_detail = FormEditRebuild(deprecated_form_id=deprecated_form.form_id)
             for case_id in affected_cases:
                 case = case_db.get(case_id)
+                previous_owner = case.owner_id
```
Comment (snopoke) [inline corehq/form_processor/backends/sql/processor.py]: looks like `case` could be `None` here

**Comment by snopoke** [comment]
Looks OK, don't really have any suggestions for improvement

**Comment by czue** [comment]
:white_check_mark:

**Pull request merged**